app/controllers/admin/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ApplicationController
     private
 
     def is_admin
-      return if current_user.admin?
+      return if current_user&.admin?
```

Comment:
Đã vào đây phải đảm bảo current user exist rồi em nên toán tử an toàn chỗ này là ko hặp lý